README.md

```diff
@@ -245,6 +247,15 @@ graphical Emacs, not for other Emacsen I may start at the command line:
   (add-hook 'after-init-hook 'server-start t)
   (add-hook 'after-init-hook 'edit-server-start t))
 ```
+In another example, we can load things conditional on the operating system:
+
+```
+(use-package exec-path-from-shell
+  :if (string-equal system-type "darwin")
```

Comment:
This is a very minor change, so IMO it should be okay to amend the commit and then `push --force` to update this PR.